src/VkBootstrap.cpp

```diff
@@ -145,13 +145,23 @@ bool check_extension_supported (
 }
 
 bool check_extensions_supported (std::vector<VkExtensionProperties> const& available_extensions,
-    std::vector<const char*> const& extension_names) {
+    std::vector<const char*> const& extension_names, std::vector<const char*> const& display_extension_names, std::vector<const char*> &display_managers, bool headless) {
 	bool all_found = true;
 	for (const auto& extension_name : extension_names) {
 		bool found = check_extension_supported (available_extensions, extension_name);
 		if (!found) all_found = false;
 	}
-	return all_found;
+	// The logic for the display extension names is slightly different, as we want to make sure
```

Comment:
Okay, I didn't finish this comment. "as we want to make sure there's at least one present"